docs/content/how-to-guides/configure-reactions/configure-drasi-result-reaction/_index.md

```diff
@@ -3,8 +3,137 @@ type: "docs"
 title: "Configure a Drasi Result Reaction"
 linkTitle: "Configure a Drasi Result Reaction"
 weight: 60
-toc_hide: true
-hide_summary: true
 description: >
     Learn how to configure a Drasi Result Reaction
----
\ No newline at end of file
+---
+
+The Drasi Result Reaction allows users to retrieve the current result set, or the result set at a specified timestamp, for a particular Continuous Query. This reaction provides an endpoint accessible via HTTP GET requests.
+
```

Comment:
We should point out that this Reaction is typically used for programmatic access to a query result. I dont think that comes across well, especially with all the curl examples. Maybe we could have examples in curl and/or javascript etc. And talk about the output of the Result reaction as an API that another program is goignt to call. What are the inputs/outputs. Maybe do a typespec spec for it.